src/Coxeter_triangulation/include/gudhi/Freudenthal_triangulation.h

```diff
@@ -0,0 +1,239 @@
+/*    This file is part of the Gudhi Library. The Gudhi library
+ *    (Geometric Understanding in Higher Dimensions) is a generic C++
+ *    library for computational topology.
+ *
+ *    Author(s):       Siargey Kachanovich
+ *
+ *    Copyright (C) 2019 Inria
+ *
+ *    Modification(s):
+ *      - YYYY/MM Author: Description of the modification
+ */
+
+#ifndef FREUDENTHAL_TRIANGULATION_H_
+#define FREUDENTHAL_TRIANGULATION_H_
+
+#include <stack>
+#include <map>
+#include <numeric> //iota
```

Comment:
Include what you use:
```suggestion
#include <vector>
#include <algorithm>  // for std::sort
#include <cmath> // for std::floor
#include <numeric> // for std::iota
```